package.json

```
@@ -107,12 +113,31 @@
         "icon": "$(refresh)",
         "when": "Codacy:PullRequestStateContext == Loaded"
       },
+      {
+        "command": "codacy.pullRequests.refresh",
+        "title": "Refresh Pull Requests",
+        "category": "Codacy commands",
+        "icon": "$(refresh)",
+        "when": "Codacy:RepositoryManagerStateContext == Loaded"
+      },
       {
         "command": "codacy.pr.openSummary",
         "title": "Open Pull Request Summary",
         "category": "Codacy commands",
         "when": "Codacy:PullRequestStateContext == Loaded"
       },
+      {
+        "command": "codacy.pr.checkout",
+        "title": "Checkout Pull Request",
```

Although the command is "checkout", the verb here has a space:

```suggestion
        "title": "Check Out Pull Request",
```